.huskyrc

```diff
@@ -1,6 +1,6 @@
 {
   "hooks": {
-    "pre-commit": "yarn lint",
-    "pre-push": "yarn lint && yarn test"
+    "pre-commit": "yarn lint && yarn test",
```

Comment:
I think it's out of the scope of current PR

Reply:
I just switched the `test` to the commit hook, so I can `-n` my commit and not run the test when I push b/c I know the tests are going to fail for now haha